Functions.Templates/Templates/HttpTrigger-CSharp-Isolated/.template.config/template.json

```diff
@@ -54,6 +54,19 @@
     "defaultName": "HttpTriggerCSharp",
     "postActions": [
         {
+            "condition": "(TargetFrameworkIdentifier == \".NETCoreApp\")",
+            "Description": "Adding Reference to Microsoft.Azure.Functions.Worker.Extensions.Http.AspNetCore Nuget package",
+            "ActionId": "B17581D1-C5C9-4489-8F0A-004BE667B814",
+            "ContinueOnError": "true",
+            "ManualInstructions": [],
+            "args": {
+                "referenceType": "package",
+                "reference": "Microsoft.Azure.Functions.Worker.Extensions.Http.AspNetCore", "version": "1.0.0",
```

Comment:
This version will need to be updated once release is cut, right?

Reply:
Should there also be a framework reference for ASP.NET Core here as well? `<FrameworkReference Include="Microsoft.AspNetCore.App" />`

Reply:
Confused by this ask - what does that mean in the context of a `template.json` file?

Reply:
I'm assuming the scenario where the customer has not included this in their project file otherwise. Since it's a requirement here, I'm wondering if it makes sense to add an additional reference. Does `referenceType` allow for a value like "framework" or something?

Reply:
It would appear that it does not... https://github.com/dotnet/templating/wiki/Post-Action-Registry#add-a-reference-to-a-project-file
"package" or "project", unfortunately

(Also hey look - in-proc is the first example there)